src/main/resources/edu/cornell/kfs/module/purap/cu-purap-resources.properties

```diff
@@ -3,4 +3,11 @@ iwnt.note.create.reqs=Requisition edoc {0} created by {1} ({2})
 jaggaer.upload.xml.error.message=Message type '{0}' with a value of '{1}'
 jaggaer.upload.webservice.error=There was an error calling the Jaggaer upload service
 jaggaer.xml.website.error=Vendor number {0} has an invalid website URL of {1}
-jaggaer.xml.location.name.format={0} - {1}
\ No newline at end of file
+jaggaer.xml.location.name.format={0} - {1}
+jaggaer.xml.report.section.header=*********************** {0} **************************
+jaggaer.xml.report.section.footer=******************************************************
+jaggaer.xml.report.summary.file.name=Summary for {0}
+jaggaer.xml.report.summary.active.inactive.total=\t\t Total number of {0} vendors: {1}
```

Comment:
I see that the tab-prefixed messages also have a single space immediately after the tabs. Is that intentional, or should such lines be prefixed with only tabs or only spaces?

Reply:
Thanks for the clarification. I'm fine with the tabs and spaces being left in as-is.